schematic/manifest/generator.py

```diff
@@ -1399,6 +1419,9 @@ def get_manifest_with_annotations(
             (k, list(v.values())) for k, v in annotations_dict_raw.items()
         )
 
+        # Make sure that the Component column is full
+        annotations_dict["Component"] = [self.root] * max(1, len(annotations.index))
```

Comment:
Note: All changes in this PR are from running `black`, with the exception of this line and the preceding comment.